### PR TITLE
Add pytorch implementations of PSNR and SSIM

### PR DESCRIPTION
**Pull request by tmke8**
PSNR is working perfectly. SSIM has some problems.

### CODE REVIEW

**Comment by tmke8** [comment]
This is ready to be reviewed @predictive-analytics-lab/huawei-ai

File: utils/psnr.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        Args:
+            data_range: possible range of values. 255 for uint8; 1 for output of sigmoid
+        """
+        super().__init__()
+        self.scale = 1.0 / data_range**2
+
+    def forward(self, im_true, im_test):
+        err = (im_true - im_test)**2
+        mean_err = err.mean(-1).mean(-1).mean(-1)  # reduce the last three dimensions: HxWxD
+        return -10 * torch.log10(self.scale * mean_err)
```
Comment (olliethomas) [inline utils/psnr.py:22]: should this be negative?

Comment (olliethomas) [reply utils/psnr.py:22]: I mean, I can see the test, so I'm sure it's right. Just trying to work out why

https://en.wikipedia.org/wiki/Peak_signal-to-noise_ratio

Comment (olliethomas) [reply utils/psnr.py:22]: 😅 just discovered the submit comments button

Comment (tmke8) [reply utils/psnr.py:22]: `log(1/x)` is equivalent to `-log(x)`

**Pull request merged**